### PR TITLE
JS Alert When Name/Title is Empty During Email/Optin Automation

### PR DESCRIPTION
**Pull request by ShoboySnr**
Show JS alert if title or name is empty

### CODE REVIEW

File: src/assets/js/admin/new-optin-campaign.js [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
             if (!optin_campaign_title_obj.val()) {
                 optin_campaign_title_obj.addClass('mailoptin-input-error');
+                alert('Title of the Optin Campaign is required');
```
Comment (w3guy) [inline src/assets/js/admin/new-optin-campaign.js:13]: This string is not translatable. Go to wp-content/plugins/mailoptin/Core/src/RegisterScripts.php Line 237. Add the string to the js object and then call the string via `mailoptin_globals.objectItemName`.

Read more on wp_localize_script here https://developer.wordpress.org/reference/functions/wp_localize_script/

Comment (ShoboySnr) [reply src/assets/js/admin/new-optin-campaign.js:13]: okay

**Comment by ShoboySnr** [comment]
Can you confirm if it works as requested.

**Comment by w3guy** [comment]
I will when I can.

**Comment by ShoboySnr** [comment]
Okay sure..

**Pull request merged**